src/components/Cohorts/Pagination.tsx

```diff
@@ -0,0 +1,32 @@
+import { Grid } from '@mui/material';
+import Pagination from '@mui/material/Pagination';
+
+const BasicPagination = ({
+  pages,
+  currentPage,
+  setCurrentPage,
+}: {
+  pages: string;
+  currentPage: string;
+  setCurrentPage: (page: string) => void;
+}) => {
+  // eslint-disable-next-line no-undef
+  const handlePageChange = (event: React.ChangeEvent<unknown>, page: number) => {
```

Comment:
Why disable `eslint`
+ Why add `event` as a parameter if you didn't use it?

Reply:
the event parameter in the handlePageChange function, it is included to handle the React.ChangeEvent that is triggered when the page changes in the Pagination component. Although the event parameter is not directly used within the function body, it is still necessary to include it in the function signature to maintain compatibility with the onChange event handler of the Pagination component. If the event parameter is not included, it may lead to type errors or warnings in development tools.